packages/components/src/components/themed-scrollbars/themed-scrollbars.scss

```diff
@@ -34,6 +34,27 @@
             overflow-x: overlay;
         }
     }
+    &--p2p-cashier-wrapper {
```

Comment:
Same here, rather than adding app-specific rules to this shared components package, either try to make the names more generic so other apps can also use them or add the logic to the app itself.

Reply:
Still the same, @kseniya-deriv.

Reply:
it's outdated, I changed several days ago